homeassistant/components/flo/device.py

```diff
@@ -42,7 +42,11 @@ async def _async_update_data(self):
         try:
             async with timeout(10):
                 await asyncio.gather(
-                    *[self._update_device(), self._update_consumption_data()]
+                    *[
+                        self.send_presence_ping(),
```

Comment:
Does this need to be executed first in order for these other calls to work? `asyncio.gather` makes no guarantee about call order.

Reply:
So I debated this myself (and initially coded it that way with this new call before the gather)… given that this runs every 30 seconds I don’t think splitting the calls out makes sense. At most the data would be 1 update cycle old… is that worth stacking the calls and making them run serially? I didn’t think so but I’m open to other opinions.

Reply:
Got it. Let's roll with what you have; we can always adjust later.